### PR TITLE
fix(Sitecore): replace HasFiles with HasFilesInDir for user.json file…

### PR DESCRIPTION
**Pull request by dorokhin-bohdan**
… (#5)

* fix(Sitecore): replace HasFiles with HasFilesInDir for user.json file

* fix(Sitecore): remove useless test files

---------

### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md].
- [x] The commit message follows the [conventional commits][cc] guidelines.
- [x] Tests for the changes have been added (for bug fixes / features).
- [x] Docs have been added/updated (for bug fixes / features).

### Description

The Sitecore segment was not displayed in a properly configured directory.
The `HasFiles` method returns `false` for `path.Join(sitecoreFolderName, userFileName)` argument while `HasFilesInDir` returns `true` (expected result). 

image

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: src/segments/sitecore.go [hunk 1]
```diff
@@ -33,13 +34,16 @@ type UserConfig struct {
...
+	if !s.env.HasFiles(sitecoreFileName) || !s.env.HasFilesInDir(sitecoreFolderName, userFileName) {
+		s.env.Debug("The Sitecore CLI configuration files were not found")
 		return false
 	}
 
 	var userConfig, err = getUserConfig(s)
 
 	if err != nil {
+		dbgMsg := fmt.Sprintf("Unable to read '%s' file. %s", userFileName, err.Error())
+		s.env.Debug(dbgMsg)
```
Comment (JanDeDobbeleer) [inline src/segments/sitecore.go]: This needs to be `s.env.Err(err)`

File: src/segments/sitecore.go [hunk 2]
```diff
@@ -48,6 +52,7 @@ func (s *Sitecore) Enabled() bool {
 	displayDefault := s.props.GetBool(properties.DisplayDefault, true)
 
 	if !displayDefault && s.EndpointName == defaultEnpointName {
+		s.env.Debug("The displaying of the default environment is turned off")
```
Comment (JanDeDobbeleer) [inline src/segments/sitecore.go]: Logs need to start with a lowercase

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: src/segments/sitecore.go [hunk 3]
```diff
@@ -33,13 +33,16 @@ type UserConfig struct {
...
-	if !s.env.HasFiles(sitecoreFileName) || !s.env.HasFiles(path.Join(sitecoreFolderName, userFileName)) {
+	if !s.env.HasFiles(sitecoreFileName) || !s.env.HasFilesInDir(sitecoreFolderName, userFileName) {
+		s.env.Debug("sitecore cli configuration files were not found")
 		return false
 	}
 
 	var userConfig, err = getUserConfig(s)
 
 	if err != nil {
+		s.env.Debug("unable to read 'user.json' file")
```
Comment (JanDeDobbeleer) [inline src/segments/sitecore.go]: this is absolete, the error will report where we are in the code as well.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
One last change and we're good to go.

File: src/segments/sitecore.go (hunk 3 above)
Comment (dorokhin-bohdan) [reply src/segments/sitecore.go]: thanks, for the review

The log was removed.

**Review by JanDeDobbeleer — APPROVED** [review]

**Pull request merged**